cyber-resilience-sig/coordination/enisa/deliverable-2-6.md

```diff
@@ -0,0 +1,67 @@
+---
+SIG: Cyber Resilience SIG
+Document Type: Deliverable
+Number: 2.6
+Status: ⚠️ Draft
+Group: ENISA
+Date: 2025-06-14
+---
+
+# Feedback to call for evidence on Cybersecurity Act Revision
+
+## Context
+
+Provide feedback to the [EU Commission][]'s [open call for evidence](https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/14578-The-EU-Cybersecurity-Act_en) as part of its [initiative to revise the Cybersecurity Act (CSA)](https://digital-strategy.ec.europa.eu/en/news/commission-opens-consultation-revising-eu-cybersecurity-act). Urge for a stronger technical and strategic role for ENISA in light of the [recent funding issues of the US CVE program](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures#2025_funding_issues) and the increased importance of vulnerability handling in EU legislation.
+
+## Goals
+
+* Urge the EU Commission to increase ENISA's mandate regarding CVE handling
+* Avoid disproportionate reliance on U.S. government funding for CVEs
+
+## Draft feedback
+
+> [!NOTE]
+> Feedback must be provided in a comment box (so no formatting option). Text length is limited to 4,000 characters.
+
+> In light of the and wider need for harmonisation of vulnerability disclosure requirements across several EU regulations (e.g. CRA, NIS2, DORA, AI act) as well as further sharing of supply-chain best-practices, we wish to share and reiterate the importance of the Common Vulnerabilities and Exposures (CVE) system and a stronger technical and strategic role for ENISA given its growing international objective:
+> 
+> We recognise the importance of increasing EU resources to ENISA as part of CSA review. Notably, ENISA’s role as a CVE Numbering Authority (CNA) is important and industry welcomes ENISA’s goal to become a Root CNA.
+> 
+> We also believe that ENISA should step up as a CNA of Last Resort (for Europe, regardless of report or vendors jurisdiction). Secondly we believe that ENISA should contribute systematically and long term to the CVE system as a whole; in general, and in particular around governance, establishing industry best practices and driving improvements. This also includes a pro-active, operational role, in ensuring that coordinated disclosure processes are followed and that the data is of sufficient quality to be useful in a European setting in general and for compliance with, for example, NIS2 and the CRA in particular. This may, at times, include helping the industry by annotating records, defining vocabulary or by re-publishing advisories after normalisation.
+> 
+> MITRE’s role as Secretariat for CVE program plays a key role to provide a resource to boost supply chain resilience. While its recent publicity and uncertainty in funding are regrettable, we are pleased that this matter has now been partially resolved.  
```

Comment:
Proposed change to address @rjb4standards's [comment](https://github.com/orcwg/orcwg/pull/127#discussion_r2151945494)

```suggestion
> MITRE’s role as Secretariat for CVE program plays a key role to provide a resource to boost supply chain resilience. While its recent publicity and uncertainty in funding are regrettable, we are pleased that this matter has been temporarily resolved.  
```